### PR TITLE
Account settings tests 194

### LINKED ISSUE #194: Add tests to change password view
The change password view need tests.

### LINKED ISSUE #201: Add missing splash screen
Add missing splash screen

### LINKED ISSUE #208: Bugs: Style Issues
Login screen are not mobile-responsive:

Screen Shot 2022-08-31 at 3 44 54 PM

Add a bottom border to mobile menu dropdown:
Screen Shot 2022-08-31 at 3 40 34 PM

Align logo on footer with top nav:
Screen Shot 2022-08-31 at 3 40 15 PM

### PR DESCRIPTION
**Pull request by Carla-Moz**
This PR  adds the missing splash screen and launch text card:
Screen Shot 2022-09-02 at 5 24 03 PM

Makes the login cards mobile responsive:
Screen Shot 2022-09-02 at 5 20 42 PM

And aligns the footer and top nav Rally logos. 

Closes #194 
Closes #201 
Closes #208

### CODE REVIEW

File: web-platform/website/src/components/navigation-bar/MobileMenu.tsx [hunk 1]
```diff
@@ -159,7 +159,9 @@ const styles = {
           ".dropdown-menu": {
             transform: "none",
             width: "100%",
+            borderRadius: "4px",
```
Comment (thomik-corp) [inline web-platform/website/src/components/navigation-bar/MobileMenu.tsx:162]: Lets use Spacing enum values/

File: web-platform/website/src/components/pages/login/LoginButton.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { Button, ButtonProps, Col, Container, Row } from "reactstrap";
 import { style } from "typestyle";
 
-import { Spacing } from "../../../styles/Spacing";
+import { Spacing, createResponsiveStyle, ScreenSize } from "../../../styles";
```
Comment (thomik-corp) [inline web-platform/website/src/components/pages/login/LoginButton.tsx:4]: Running npm run lint should fix ordering here.

File: web-platform/website/src/components/pages/login/LoginButton.tsx [hunk 3]
```diff
@@ -40,5 +40,9 @@ const styles = {
     paddingBottom: Spacing.Medium,
     borderWidth: 2,
     fontWeight: 700,
-  }),
+  },
+  createResponsiveStyle(ScreenSize.ExtraSmall, {
+    padding: "10px"
```
Comment (thomik-corp) [inline web-platform/website/src/components/pages/login/LoginButton.tsx:45]: I've been avoiding custom values. Can we use Spacing enums that are close to this value?

File: web-platform/website/src/components/pages/login/LoginPageContent.tsx [hunk 4]
```diff
@@ -62,13 +84,18 @@ const styles = {
           maxWidth: "100%",
           height: "auto",
         },
+        ".login-col": {
+          marginRight: "3rem"
```
Comment (thomik-corp) [inline web-platform/website/src/components/pages/login/LoginPageContent.tsx:88]: Same here and below. IMO using custom values defeats the purpose of having enums that make the site more standardized. So 3 rem translates to 3 * 16 = 48px which translates to 1.2 * Spacing.xxxLarge I believe.

**Review by thomik-corp — APPROVED** [review]

File: web-platform/website/src/components/pages/login/LoginPageContent.tsx (hunk 4 above)
Comment (Carla-Moz) [reply web-platform/website/src/components/pages/login/LoginPageContent.tsx:88]: All right, will keep in mind moving forward.

**Pull request merged**